pkg/server/handler_helpers.go

```diff
@@ -33,6 +33,15 @@ func RespondWithJson(w http.ResponseWriter, obj interface{}) {
 	}
 }
 
+func RespondWithK8s(w http.ResponseWriter, obj interface{}) {
+	err := json.NewEncoder(w).Encode(obj)
```

Comment:
Can we do this in default RespondWithJson? nesting under `data` seems not necessary.

Reply:
Yes, I agree.

My original thought was that each response would have a "status" and a "data" to make it uniform.

Reply:
Do you want to update RespondWithJson to match this in follow up patch?